chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/surveillance/SurveillanceValidator.java

```diff
@@ -349,4 +355,26 @@ public void validateSurveillanceNonconformities(Surveillance surv) {
 			}
 		}
 	}
+	
+	public void validateSurveillanceAuthority(Surveillance surv) {
```

Comment:
Right now, the SurveillanceController has an updateNullAuthority() method that updates the authority depending on the User object's role. I'm confused because you say that the Controller works pretty much as I have it (it's checking the User roles and updating them there), but then you're advocating for checking roles in the manager. 

Again, I'm doing the below functionality in the SurveillanceValidator.validateSurveillanceAuthority (which gets called from the validate(Surveillance surv) method):

> The Manager update method is slightly more complicated only in that you need to check that the user performing the update isn't ACB_ADMIN if the Authority in the existing surveillance db row is ONC_ADMIN (if ONC created the surveillance then ACB can't be the one to update it) - basically just check that the user performing the update is allowed to.

> 

Isn't it better/faster to check the the user's roles and throw a ValidationException before getting to the manager?